R/format.R

```diff
@@ -109,7 +109,7 @@ formatDate = function(table, columns, method = 'toDateString', params = NULL) {
     colnames = base::attr(x, 'colnames', exact = TRUE)
     rownames = base::attr(x, 'rownames', exact = TRUE)
     if (is.null(params)) params = list()
-    cols = as.character(name2int(columns, colnames, rownames))
+    cols = sprintf("%d", name2int(columns, colnames, rownames))
```

Comment:
Yes, this is more robust, but will still have problems if a column index is too big (bigger than 2^31). Again, I think it is going to be really rare, so that is fine to me.